lnbits/app.py

```diff
@@ -58,6 +62,18 @@ def create_app() -> FastAPI:
             "url": "https://raw.githubusercontent.com/lnbits/lnbits/main/LICENSE",
         },
     )
+    limiter = Limiter(key_func=lambda request: request.client.host, default_limits=[settings.lnbits_rate_limit + "/hour"])
```

Comment:
lnbits_rate_limit + "/hour" can be replaced by a setting that includes the `/hour` (and can be changed by the user)